sklearn/tests/test_kernel_approximation.py

```diff
@@ -245,13 +244,3 @@ def logging_histogram_kernel(x, y, log):
              n_components=(n_samples - 1),
              kernel_params={'log': kernel_log}).fit(X)
     assert_equal(len(kernel_log), n_samples * (n_samples - 1) / 2)
-
-    def linear_kernel(X, Y):
-        return np.dot(X, Y.T)
-
-    # if degree, gamma or coef0 is passed, we raise a warning
-    msg = "Passing gamma, coef0 or degree to Nystroem"
-    params = ({'gamma': 1}, {'coef0': 1}, {'degree': 2})
-    for param in params:
-        ny = Nystroem(kernel=linear_kernel, **param)
-        assert_warns_message(DeprecationWarning, msg, ny.fit, X)
```

Comment:
This should be,
```py
with pytest.raises(ValueError, match=msg):
    ny.fit(X)
```
as the warning got transformed to an error..

Reply:
why does this pass then? hm...

Reply:
right.